pkg/storage/watch/update/update.go

```diff
@@ -0,0 +1,10 @@
+package update
+
+import (
+	meta "github.com/weaveworks/ignite/pkg/apis/meta/v1alpha1"
+)
+
+type Update struct {
```

Comment:
godoc

Reply:
Done in https://github.com/weaveworks/ignite/pull/242.